ExpSwift/Classes/Exp.swift

```diff
@@ -1015,6 +1036,77 @@ public func respond(_ params:[String:Any]) -> Promise<Message>{
     }
 }
 
+/**
+ Get Current User
+ @param params.
+ @return Promise<User>.
+ */
+public func getCurrentUser() -> Promise<User>{
+    return Promise { fulfill, reject in
+        Alamofire.request(Router.getCurrentUser()).validate()
+            .responseObject { (response: DataResponse<User>) in
+                switch response.result{
+                case .success(let data):
+                    fulfill(data)
+                case .failure(let error):
+                    return reject(error)
+                }
+        }
+    }
+}
+
+/**
+ Get Current User
+ @param params.
+ @return Promise<User>.
+ */
+public func getToken(_ options:[String:Any]) -> Promise<Auth>{
+    return Promise { fulfill, reject in
+        Alamofire.request(Router.getToken(options)).validate()
+            .responseObject { (response: DataResponse<Auth>) in
+                switch response.result{
+                case .success(let data):
+                    fulfill(data)
+                case .failure(let error):
+                    return reject(error)
+                }
+        }
+    }
+}
+
+
+/**
+ Get Current User
+ @param params.
+ @return Promise<User>.
+ */
+public func getUser(_ host:String, token:String) -> Promise<User>{
+    expLogging("EXP GET USER  : \(token)")
+    tokenSDK = token
+    hostUrl = host
+    return Promise { fulfill, reject in
+        Alamofire.request(Router.getCurrentUser()).validate()
+            .responseObject { (response: DataResponse<User>) in
+                switch response.result{
+                case .success(let data):
+                    fulfill(data)
+                case .failure(let error):
+                    return reject(error)
+                }
+        }
+    }
+}
+
+/**
+ Starth with Auth 
+ @param options.
+ @return Promise<Auth>.
+ */
+func startAuth(_ auth:Auth) ->Promise<User>{
+    return getCurrentUser()
```

Comment:
what is this function doing? it seems like it doesn't need the auth param, is there a reason?